source/_docs/configuration/splitting_configuration.markdown

```diff
@@ -27,11 +27,13 @@ homeassistant:
   customize: !include customize.yaml
 ```
 
-Note that each line after `homeassistant:` is indented two (2) spaces. Since the configuration files in Home Assistant are based on the YAML language, indentation and spacing are important. Also note that seemingly strange entry under `customize:`.
+Note that each line after `homeassistant:` is indented two (2) spaces. Since the configuration files in Home Assistant are based on the YAML language, indentation and spacing are important.
 
-`!include filename.yaml` is the statement that tells Home Assistant to insert the contents of `filename.yaml` at that point. This is how we are going to break a monolithic and hard to read file (when it gets big) into more manageable chunks.
+Also, note the seemingly strange entry after [`customize:`](/docs/configuration/customizing-devices).
```

Comment:
If anything, we should look into replacing `customize` here.

In the years, things changed a lot. And customize is actually not something people should use anymore (discouraged).

Reply:
Swapping it for an example of including a file for another purpose?